build/config/compiler/BUILD.gn

```diff
@@ -204,6 +204,13 @@ config("strict_warnings") {
     "-Wvla",
   ]
 
+  if (target_os == "webos") {
```

Comment:
This needs to say current_os not target_os.

The way that target_os is being used in this PR is not correct. It's just setting the default toolchain, and the initial value for current_os in the default toolchain. They have the same value space. We shouldn't have target_os = "webos" set current_os = "linux".

Reply:
Thank you for your review, Michael.
Let me fix this by using 'current_os == "webos"'.

Reply:
Dear Michael,
Your comment is fixed on below commit.
https://github.com/project-chip/connectedhomeip/pull/13292/commits/332aeb0e597be919476b0ac4350c53b72d096726

I fixed the build script to have the initial value of current_os as 'webos'.

Now, branching the current_os == "webos" works on build/config/compiler/BUILD.gn
Let me know if this is the right fix.
Thank you.